test/TBD/multimodule-implied-objc.swift

```diff
@@ -0,0 +1,47 @@
+// REQUIRES: VENDOR=apple
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/cache)
+// RUN: split-file %s %t
+
+/// Create Swift modules to import.
+// RUN: %target-swift-emit-module-interface(%t/IndirectMixedDependency.swiftinterface) \
+// RUN:   %t/IndirectMixedDependency.swift -I %t
+// RUN: %target-swift-emit-module-interface(%t/SwiftDependency.swiftinterface) \
+// RUN:   %t/SwiftDependency.swift -I %t
+
+// Generate TBD file.
+// RUN: %target-swift-frontend %t/Client.swift -emit-ir -o/dev/null \
+// RUN: -parse-as-library -module-name test -validate-tbd-against-ir=missing \
+// RUN: -emit-tbd -emit-tbd-path %t/client.tbd \
```

Comment:
Should these lines be indented the same way the others are to indicate they are continuations of the previous lines?